community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/TransactionStateMachine.java

```diff
@@ -313,6 +331,10 @@ private MutableTransactionState( AuthenticationResult authenticationResult )
 
     interface SPI
     {
+        void awaitUpToDate( long oldestAcceptableTxId, int timeout, TimeUnit timeoutUnit ) throws TransactionFailureException;
```

Comment:
Done, in https://github.com/neo4j/neo4j/pull/7799/commits/ed89b68fe264a87d8b72b6dc19ed4efd07b66dc1